core/src/main/kotlin/org/jetbrains/kotlinx/dataframe/api/describe.kt

```diff
@@ -25,35 +28,132 @@ public interface ColumnDescription {
     public val mean: Double
     public val std: Double
     public val min: Any
+    public val p25: Any
     public val median: Any
+    public val p75: Any
     public val max: Any
 }
 
+/**
+ * ### Summary Metrics:
+
+ * - **`name`** — The name of the column.
+ * - **`path`** — path to the column (for hierarchical `DataFrame`)
+ * - **`type`** — The data type of the column (e.g., Int, String, Boolean).
+ * - **`count`** — The total number of non-null values in the column.
+ * - **`unique`** — The number of unique values in the column.
+ * - **`nulls`** — The count of null (missing) values in the column.
+ * - **`top`** — The most frequently occurring value in the column.
+ * - **`freq`** — The frequency of the most common value.
+ * - **`mean`** — The arithmetic mean (only for numeric columns).
+ * - **`std`** — The standard deviation (only for numeric columns).
+ * - **`min`** — The minimum value in the column.
+ * - **`p25`** — The 25th percentile value (first quartile).
+ * - **`median`** — The median value (50th percentile / second quartile).
+ * - **`p75`** — The 75th percentile value (third quartile).
+ * - **`max`** — The maximum value in the column.
+ *
+ * For non-numeric columns, statistical metrics
+ * such as `mean` and `std` will return `null`. If a column is not [Comparable],
+ * percentile values (`min`, `p25`, `median`, `p75`, `max`) will also return `null`.
+ */
+internal interface SummaryMetrics
```

Comment:
Nice :D

Reply:
You can mark this documentation interface as `@ExcludeFromSources`, just like the `interface Describe` below, since their contents are only included in other places.

Reply:
oh btw, probably link to `ColumnDescription` so users can click on it and explore the DataSchema result.